messages/run-dfa.js

```diff
@@ -6,17 +6,17 @@ module.exports = {
 	"flags": {
 		"formatDescription": "format of results",
 		"formatDescriptionLong": "Specifies output format with results written directly to the console.",
-		"ignoreparseerrorsDescription": "ignore compilation failures in scanned files",
+		"ignoreparseerrorsDescription": "ignore compilation failures in scanned files (default: false). Alternatively, set value using environment variable `SFGE_IGNORE_PARSE_ERRORS`",
 		"ignoreparseerrorsDescriptionLong": "ignore compilation failures in scanned files. Inherits value from SFGE_IGNORE_PARSE_ERRORS env-var if set.",
-		"normalizesevDescription": "A normalized severity 1 (high), 2 (moderate), and 3 (low) is returned in addition to the engine specific severity",
+		"normalizesevDescription": "return normalized severity in addition to the engine specific severity",
 		"normalizesevDescriptionLong": "A normalized severity 1 (high), 2 (moderate), and 3 (low) is returned in addition to the engine specific severity. For the html option, the normalized severity is displayed instead of the engine severity",
 		"outfileDescription": "location of output file",
 		"outfileDescriptionLong": "Write output to a file.",
-		"projectdirDescription": "Root project directory",
+		"projectdirDescription": "project's root directory, or the directory where the target location resides along with the remaining class files it references",
```

Comment:
We also look at VF pages/components. Can it be "the root folder of each project being scanned. Must contain all Apex classes and VF pages/components in the project", or something like that?